api/src/lib/index.js

```diff
@@ -17,24 +17,30 @@ export const lastEntry = (array) => {
     nextDate.setUTCSeconds(next.created)
     return firstDate - nextDate
   })
-  return latest[latest.length-1]
+
+  return latest[latest.length - 1]
 }
 
-export const handleStripeWebhooks = (event, context, webhooksObj = {}, secure = true) => {
+export const handleStripeWebhooks = async (event, context, webhooksObj = {}, secure = true) => {
```

Comment:
We weren't awaiting the user's response to webhooks; seemed like a bug, especially if we expect them to make prisma calls